src/Models/ApplicantModel.php

```diff
@@ -72,6 +69,10 @@ public function storeApplicant(array $applicant)
         $result = $query->execute();
         if ($result) {
             $id = $this->db->lastInsertId();
+            foreach ($applicant['cohort'] as $cohortId) {
+                $query2 = $this->db->prepare('INSERT INTO `course_choice` (`coursesid`, `applicantsid`) VALUES (?,?)');
```

Comment:
can we have a different variable name instead of `$query2`, something more descriptive, like `courseChoiceQuery`?